main.py

```diff
@@ -11,7 +11,48 @@
 from napps.kytos.of_core.flow import FlowFactory
 
 from .exceptions import InvalidCommandError
-from .settings import CONSISTENCY_INTERVAL, FLOWS_DICT_MAX_SIZE
+from .settings import (CONSISTENCY_COOKIE_EXCEPTION_RANGE,
+                       CONSISTENCY_INTERVAL,
+                       CONSISTENCY_TABLE_ID_EXCEPTION_RANGE,
+                       FLOWS_DICT_MAX_SIZE)
+
+
+def _validate_range(values):
+    """Check if the consistency exception range is valid."""
+    instance = int
+    if len(values) != 2:
+        msg = f'The tuple must have 2 items, not {len(values)}'
+        raise ValueError(msg)
+    first, second = values
+    if second < first:
+        msg = (f'The first value is bigger than the second: {values}')
+        raise ValueError(msg)
+    if not isinstance(first, instance) or not isinstance(second, instance):
+        msg = 'The elements of the range must be of the class {instance}'
+        raise TypeError(msg)
+
+
+def _valid_consistency_exceptions(consistency_exception_list):
+    """Check that the list of consistency exceptions is well formatted.
+
+    Returns True if the consistency exception list is well formatted or False
+    otherwise.
+    """
+    msg = ('The list of consistency exception is not well formatted'
+           ', it will be ignored: %s')
+    for consistency_exception in consistency_exception_list:
+        if isinstance(consistency_exception, tuple):
```

Comment:
Suggestion: "exception" is too related to Python's exceptions. Maybe "ignore_range", or "exclude_range" would be better. I vote for ignore.